lib/core/filtering/simple_rules/validation/simple_rules_schema.rb

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+# frozen_string_literal: true
+
+require 'core/filtering/simple_rules/simple_rule'
+
+module Core
+  module Filtering
+    module SimpleRules
+      module Validation
+        ALLOWED_VALUE_TYPES = ->(rule_value) { rule_value.is_a?(String) || rule_value.is_a?(Integer) || rule_value.is_a?(TrueClass) || rule_value.is_a?(FalseClass) }
+        MATCH_ALL_REGEX_NOT_ALLOWED = ->(simple_rule) { !(simple_rule['rule'] == Core::Filtering::SimpleRule::Rule::REGEX && (simple_rule['value'] == '(.*)' || simple_rule['value'] == '.*')) }
```

Comment:
will this complain about the default rule?

Reply:
It didn't during testing. I guess the default rule is not sent for validation? Is the ID of the default rule always the same? If yes I could add a check for that.